IFC4x3/Properties/i/Inductance_26NOHc5AHAsRdIVv$vqQ_g/Documentation.md

```diff
@@ -0,0 +1 @@
+The inductance of the inductor
```

Comment:
Measure of the Inductance

Reply:
Done, Commit : 0bb5ae0d772bc5a99c0bea0886d3f8ed41fd2f5d